Logger/Logger.csproj

```
@@ -1,8 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>10.0</LangVersion>
+	<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+	<TargetFramework>net8.0</TargetFramework>
+	<LangVersion>12.0</LangVersion>
+	<Nullable>enable</Nullable>
+	<EnableNETAnalyzers>True</EnableNETAnalyzers>
+	<Nullability>True</Nullability>
+	<CodeAnalysisTreatWarningAsErrors>True</CodeAnalysisTreatWarningAsErrors>
+	<EnforceCodeStyleInBuild>True</EnforceCodeStyleInBuild>
+	<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
```

These should be lowercase
```suggestion
	<CodeAnalysisTreatWarningAsErrors>true</CodeAnalysisTreatWarningAsErrors>
	<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
	<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```